config/webpack.base.config.js

```diff
@@ -0,0 +1,44 @@
+const webpack = require("webpack");
+const merge = require("webpack-merge");
+
+const CopyWebpackPlugin = require("copy-webpack-plugin");
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+
+module.exports = env => {
+  const { PLATFORM, VERSION } = env;
+  return merge([
+      {
+        module: {
+          rules: [
+            {
+              test: /\.js$/,
+              exclude: /node_modules/,
+              use: {
+                loader: 'babel-loader'
+              }
+            },
+            {
+              test: /\.scss$/,
+              use: [
+                PLATFORM === 'production' ? MiniCssExtractPlugin.loader : 'style-loader',
+                'css-loader',
+                'sass-loader'
+              ]
+            }
+          ]
+        },
+        plugins: [
+          new HtmlWebpackPlugin({
+            template: './src/index.html',
+            filename: './index.html'
+          }),
+          new CopyWebpackPlugin([ { from: 'src/static' } ]),
```

Comment:
Do not use this plugin if you configure rules for images

Reply:
Done. Removed CopyWebpackPlugin